Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -374,6 +374,48 @@ namespace O3DE::ProjectManager
         }
     }
 
+
+    void GemItemDelegate::DrawPlatformText(QPainter* painter, const QRect& contentRect, const QFont& standardFont, const QModelIndex& modelIndex) const
+    {
+        const GemInfo::Platforms platforms = GemModel::GetPlatforms(modelIndex);
+        
+        auto xbounds = CalcColumnXBounds(HeaderOrder::Name);
+        const int startX = s_platformTextleftMarginCorrection + xbounds.first;
+        QFont platformFont(standardFont);
+        platformFont.setPixelSize(s_featureTagFontSize);
+        platformFont.setBold(false);
+        painter->setFont(platformFont);
+        QString platformText = "";
+
+        //If no platforms are specified, there is nothing to draw
+        if(platforms == 0)
+        {
+            return;
+        }
+        
+        //UX prefers that we show platforms in reverse alphabetical order
+        for(int i = GemInfo::NumPlatforms-1; i >= 0; i--)
+        {
+            const GemInfo::Platform platform = static_cast<GemInfo::Platform>(1 << i);
+            if (platforms & platform)
+            {
+                const QString& singlePlatformText = GemInfo::GetPlatformString(platform);
+                if(i != GemInfo::NumPlatforms-1)
```

Comment:
Shouldn't we just be checking that platformText isn't empty? It looks like if a gem doesn't support Windows but supports other platforms then we'll be adding a comma to the start of the string.

Reply:
I just confirmed the bug occurs as you listed here. I think it would be better to switch to building a QStringList and then using join on `,`

Reply:
this will be fixed in my upcoming commit for this PR